packages/svelte/src/internal/client/dom/elements/attributes.js

```diff
@@ -221,7 +221,10 @@ export function set_custom_element_data(node, prop, value) {
 			// Don't compute setters for custom elements while they aren't registered yet,
 			// because during their upgrade/instantiation they might add more setters.
 			// Instead, fall back to a simple "an object, then set as property" heuristic.
-			setters_cache.has(node.nodeName) || customElements.get(node.tagName.toLowerCase())
+			setters_cache.has(node.nodeName) ||
+			// customElements may not be available in browser extension contexts
+			!customElements ||
```

Comment:
Is the variable defined but falsy in these cases, or is it a ReferenceError to even try to use it?

Reply:
Oh, looking at the error message in the original issue, it sounds like it is defined but `null` maybe? Interesting.

Reply:
yeah exactly